src/windows.toit

```diff
@@ -143,26 +174,62 @@ is-absolute "../foo/bar"       // False.
 ```
 */
 is-absolute path/string -> bool:
+  return is-absolute-volume path or is-absolute-unc path
+
+/**
+Path starts with a UNC volume.
+For example: `//host/share`, `//./UNC/host/share`, ...
+*/
+is-absolute-unc path/string -> bool:
   volume-name-size := volume-name-size_ path
 
   if volume-name-size == 0:
     // Path doesn't have a volume name.
     // For example: `foo/bar`, `../foo`, `\foo\bar`, ...
     return false
 
-  assert: path.size >= 2
-  if is-separator path[0] and is-separator path[1]:
-    // Path starts with a UNC volume.
-    // For example: `//host/share`, `//./UNC/host/share`, ...
-    return true
+  return is-separator path[0]
 
-  if path.size == volume-name-size:
-    // Path is just a volume name.
-    // For example: `c:`.
-    return false
+/**
+Path starts with an absolute drive indicator.
+For example: `c:\\', 'd:/`, ...
```

Comment:
If this function stays public, then it should:
- start with a 3rd person verb.
- have the examples in an `# Examples` section.

It should also have examples for failing cases, like `//host/share/foo` or `c:foo`.

Reply:
I made it private.